migrate/20210623152412_create_tables.go

```diff
@@ -0,0 +1,366 @@
+package migrate
```

Comment:
Do we need so many numbers in the migration filename schema? How the following to make it more readable:
```
YYYY-MM-DD_HH-MM-SS_<migration name>
```

Reply:
We would need to fork the library which we will likely do in the future as I'd like to rewrite it. This migration name is generated by "go run migrate.go create"